### PR TITLE
CloudWatch: Added AWS MediaStore metrics and dimensions

### PR DESCRIPTION
**Pull request by ilyastoli**
What this PR does / why we need it:
Adding AWS MediaStore metics according to: https://docs.aws.amazon.com/mediastore/latest/ug/monitor-with-cloudwatch-metrics.html
Which issue(s) this PR fixes:
No issue

Special notes for your reviewer:
Based on AWS documentation only, I didn't tried to monitor AWS MediaStore myself

### CODE REVIEW

**Comment by ilyastoli** [comment]
@jessabe ping

**Comment by ilyastoli** [comment]
@papagian ping

File: pkg/tsdb/cloudwatch/metric_find_query.go [hunk 1]
```diff
@@ -181,6 +182,7 @@ var dimensionsMap = map[string][]string{
 	"AWS/MediaConnect":      {},
 	"AWS/MediaConvert":      {"Job", "Operation", "Queue"},
 	"AWS/MediaPackage":      {"Channel", "No Dimension", "OriginEndpoint", "StatusCodeRange"},
+	"AWS/MediaStore":        {"Container name", "Object group name", "Request type"},
```
Comment (papagian) [inline pkg/tsdb/cloudwatch/metric_find_query.go:185]:
```suggestion
	"AWS/MediaStore":        {"ContainerName", "ObjectGroupName", "RequestType"},
```

**Review by papagian — CHANGES_REQUESTED** [review]
Thanks for your contribution.
Digging into [the documentation](https://docs.aws.amazon.com/mediastore/latest/ug/mediastore-ug.pdf) I concluded that the dimensions should be camel case.

File: pkg/tsdb/cloudwatch/metric_find_query.go (hunk 1 above)
Comment (ilyastoli) [reply pkg/tsdb/cloudwatch/metric_find_query.go]: @papagian are we good?

**Review by papagian — APPROVED** [review]

**Pull request merged**